dev/tools/backport-pr.sh

```diff
@@ -50,6 +50,15 @@ else
 
 fi
 
+if [[ $(git diff HEAD ${BRANCH} -- *.mli) ]]; then
```

Comment:
I think this will break if the top-level directory ever gets a .mli file, because the shell will expand *.mli.  I think you want to quote it.

Reply:
Also, I'm confused by this test.  How does it work?  And what happens if some mli files contain active characters in their filenames?  I would have tested non-emptiness of the string ( but I know single brackets better than double brackets), and I would have quoted the $(..) call.

Reply:
I would use `--quiet` instead of a bracket test.

Reply:
I inspired from the test at the end of @maximedenes' merge script but thank you for suggesting `--quiet`. It's a better way. In fact, I'm using `-exit-code` instead because I don't mind seeing the diff.